server/app.py

```diff
@@ -88,6 +88,26 @@ def send_ready_up_status(room):
         room=room
     )
 
+def check_game_complete(room):
+    ''' Checks if the game is over and sends a message that contains the winner to each client'''
+    active_players = []
+    for player in ROOMS[room]['activePlayers'].keys():
+        if not ROOMS[room]['activePlayers'][player][3]:
+            active_players.append(player)
+    active_players_set = set(active_players)
+    players_finished_set = set(ROOMS[room]['playersFinished'])
+    if players_finished_set == active_players_set:
+        # We also include the winning player name in the 'gameComplete' message.
+        winning_player = max(ROOMS[room]['activePlayers'], key=ROOMS[room]['activePlayers'].get)
+        print("THIS IS THE WINNING PLAYER:", winning_player)
+        print("THIS IS THE KEY:", ROOMS[room]['activePlayers'])
+        update_db_gameswon(winning_player)
```

Comment:
Looks great! I will add a check here to see if the room only has one player and if so, i will not update gameswon.